References: bsc#1208478
Author: Joan Torres <joan.torres@suse.com>
Subject: Don't sign the Java client

This avoids the addition of a signed file in the jar with a timestamp.
The signing was necessary for accessing the system clipboard 
when the Java client runs as an applet.
(This shouldn't harm due to the deprecation of Java applets 
was done in version 9).
Index: java/CMakeLists.txt
===================================================================
--- java/CMakeLists.txt.orig
+++ java/CMakeLists.txt
@@ -154,15 +154,6 @@ add_custom_command(OUTPUT VncViewer.jar
     com/jcraft/jsch/jce/*.class
     com/jcraft/jsch/*.class
     com/tigervnc/vncviewer/*.png
-    com/tigervnc/vncviewer/tigervnc.ico
-  COMMAND ${CMAKE_COMMAND}
-  ARGS -DJava_PATH=${Java_PATH} -DJAR_FILE=${BINDIR}/VncViewer.jar
-    -DJAVA_KEYSTORE=${JAVA_KEYSTORE}
-    -DJAVA_KEYSTORE_TYPE=${JAVA_KEYSTORE_TYPE}
-    -DJAVA_STOREPASS=${JAVA_STOREPASS}
-    -DJAVA_KEYPASS=${JAVA_KEYPASS}
-    -DJAVA_KEY_ALIAS=${JAVA_KEY_ALIAS}
-    -DJAVA_TSA_URL=${JAVA_TSA_URL}
-    -P ${SRCDIR}/cmake/SignJar.cmake)
+    com/tigervnc/vncviewer/tigervnc.ico)
 
 add_custom_target(java ALL DEPENDS VncViewer.jar)
